pip-requirements.txt

```diff
@@ -1,4 +1,4 @@
 tabulate
--e git+https://github.com/lingpy/lingpy.git@86c9f22ef347a4428ed99970c89d83fa8205dce5#diff-9584dcd0d5e720b72181112bb1b56c58#egg=lingpy
+lingpy
```

Comment:
should we specify a min-version of lingpy here?

Reply:
Yes, good idea.